package.json

```diff
@@ -10,7 +10,9 @@
     "./datastore": "./lib/datastore/index.js",
     "./discovery": "./lib/discovery/index.js",
     "./datatype": "./lib/datatype/index.js",
-    "./sqlite": "./lib/sqlite.js"
+    "./sqlite": "./lib/sqlite.js",
+    "./blob-store": "./lib/blob-store/index.js",
+    "./blob-server": "./lib/blob-server/index.js"
```

Comment:
should we also have an export for the fastify plugin itself? (located at `./lib/blob-server/fastify-plugin.js`)

Reply:
At this stage I'm not sure we need any of these exports, so ok to leave as-is for now.